content/200-concepts/100-components/02-prisma-client/080-null-and-undefined.mdx

```diff
@@ -134,3 +134,158 @@ updateUser: (parent, args, ctx: Context) => {
   })
 },
 ```
+
+## The effect of <inlinecode>null</inlinecode> and <inlinecode>undefined</inlinecode> on conditionals
+
+There are some caveats to filtering with conditionals which might produce unexpected results. When filtering with conditionals you might expect one result but receive another given how Prisma treats `nullable` values.
+
+### A high level overview
+
+
+|Operator                     | 0 filters              | 1 filter               | n filters            |
+|-----------------------------|------------------------|------------------------|----------------------|
+|<inlinecode>OR</inlinecode>  | return empty list      | validate single filter | validate all filters |
+|<inlinecode>AND</inlinecode> | return all items       | validate single filter | validate all filters |
+|<inlinecode>NOT</inlinecode> | return all items       | validate single filter | validate all filters |
+
+
+### <inlinecode>OR</inlinecode>
+
+The `OR` operator states that one or more conditions must be true for the result to return any data. It is therefore logical to assume that using the `OR`
+operator but supplying no filters, returns no results.
+
+```js
+prisma.myTable.findMany({
+    where: {
+        OR: [],
+    },
+});
+```
+
+> This is the same as passing `null`, which maps to `null` in the database. If you were to pass `undefined` that would represent no value at all and so would be the same as passing no conditions to the query.
+
+When supplying the operator with a single filter, that filter will be run and it's results returned. If multiple filters are supplied then `n` results will be returned dependant on the filters supplied.
+
+```js
+// Single filter
+prisma.myTable.findMany({
+    where: {
+        OR: [{name: 'Emelie'}],
+    },
+});
+
+// Multiple filters
+prisma.myTable.findMany({
+    where: {
+        OR: [{
+          name: 'Emelie',
+          drivesCar: true
+        }],
+    },
+});
+```
+
+See the `OR` section in the [Prisma Client reference](../../../reference/api-reference/prisma-client-reference#or) for more information about the operator.
```

Comment:
I think we can make this an inline link at the top of the section - e.g. '[`OR` operator'](link) - same goes for `AND` and `NOT`

Reply:
So the link to the reference docs for each on will be the header for each section? Would i then use the <inlinecode> component?

Reply:
Link to the reference docs the first time you mention the operator, like this:

```
The [`OR`](link-here-instead) <span class="api"></span> operator states that one or more conditions must be true for the result to return any data.
```

The link text is more relevant (the link is specifically to the `OR` operator rather than ref docs in general) + cuts out a sentence.